packages/packages/v1-adapters/src/dispatchers/__tests__/index.test.ts

```diff
@@ -102,6 +105,21 @@ describe( '@elementor/v1-adapters/dispatchers', () => {
 			.rejects
 			.toEqual( '`$e.route()` is not available' );
 	} );
+
+	it( 'should determine if a route is active', () => {
+		// Arrange.
+		const route = 'test/route';
+
+		( window as any ).$e.routes.isPartOf.mockReturnValue( true );
+
+		// Act.
+		const result = isRouteActive( route );
+
+		// Assert.
+		expect( result ).toEqual( true );
+		expect( ( window as any ).$e.routes.isPartOf ).toHaveBeenCalledTimes( 1 );
```

Comment:
Why do you need it?

Reply:
to make sure that I actually call the underlying $e.routes API properly